helmfile/charts/networkpolicy-workload/templates/cert-manager/resolver.yaml

```diff
@@ -13,6 +13,5 @@ spec:
     - ports:
       - port: 8089
         protocol: TCP
----
```

Comment:
I think you need this, otherwise it will be templated as one single manifest

Reply:
did you comment on this when the manifests were not moved into sub directories in templates?